flutter/lib/benchmark/state.dart

```diff
@@ -266,13 +266,15 @@ class BenchmarkState extends ChangeNotifier {
       if (_aborting) break;
       if (!_store.submissionMode) continue;
 
-      final accuracyResult = await runBenchmark(benchmark, false,
+      final accuracyResult = await runBenchmark(benchmark, true,
           backendInfo.settings.commonSetting, backendInfo.libPath);
       _updateProgress(doneCounter * doneMultiplier / activeBenchmarks.length);
       doneCounter++;
 
       benchmark.accuracyModeResult = BenchmarkResult(
-          throughput: accuracyResult.result.throughput,
+          // loadgen doesn't calculate ltency for accuracy mode benchmarks
+          // so throughput is infinity which is not a valid JSON numeric value
+          throughput: 0.0,
```

Comment:
A condition check would be better in case `accuracyResult.result.throughput` is valid in the feature.

Reply:
Done